azurerm/resource_arm_storage_account.go

```diff
@@ -963,3 +972,124 @@ func flattenAzureRmStorageAccountIdentity(identity *storage.Identity) []interfac
 
 	return []interface{}{result}
 }
+
+func flattenAzureRmStorageAccountPrimaryEndpoints(d *schema.ResourceData, primary *storage.Endpoints, acctName *string, acctKey *string) error {
```

Comment:
There are a lot of arguments here, maybe this should be broken up/not a separate function